src/Services/UsetTwoFaService.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Services;
+
+use Blumilk\Meetup\Core\Models\User;
+use Blumilk\Meetup\Core\Models\User2FaCode;
+use Blumilk\Meetup\Core\Notifications\TwoStepVerificationNotification;
+use Blumilk\Meetup\Core\Services\Authentication\UserLoginService;
+use Illuminate\Auth\AuthenticationException;
+
+class UsetTwoFaService
+{
+    public function __construct(
+        protected UserLoginService $loginService,
+    ) {}
+
+    public function generateCode(User $user): void
+    {
+        $code = rand(100000, 999999);
```

Comment:
Why this block of code is duplicated between two classes?

Reply:
I couldn't call up the one service on the other service. now it's not duplicated